StockCount/de.lproj/Main.strings

```diff
@@ -0,0 +1,60 @@
+
+/* Class = "UILabel"; text = "Vibrate when scans"; ObjectID = "0P8-vc-Bpg"; */
+"0P8-vc-Bpg.text" = "Vibration beim Scan";
+
+/* Class = "UILabel"; text = "Comma, space and quantity"; ObjectID = "2Pn-fZ-VTh"; */
+"2Pn-fZ-VTh.text" = "Komma, Leerstelle und Stückzahl";
+
+/* Class = "UILabel"; text = "1234560D, 1"; ObjectID = "2sy-Hg-6FH"; */
+"2sy-Hg-6FH.text" = "1234560D, 1";
+
+/* Class = "UILabel"; text = "Start new scan in a new line"; ObjectID = "7sb-Ke-Dii"; */
+"7sb-Ke-Dii.text" = "Neuen Scan in neuer Zeile starten";
+
+/* Class = "UILabel"; text = "Automatically add quantity"; ObjectID = "ARM-f8-e4D"; */
+"ARM-f8-e4D.text" = "Automatische Stückzahlzufügung";
+
+/* Class = "UINavigationItem"; title = "All Files"; ObjectID = "CEW-ZG-CHd"; */
+"CEW-ZG-CHd.title" = "Alle Dateien";
+
+/* Class = "UIBarButtonItem"; title = " "; ObjectID = "CNX-Hw-bwv"; */
+"CNX-Hw-bwv.title" = " ";
+
+/* Class = "UILabel"; text = "Socket Mobile\nVersion 0.8"; ObjectID = "Cmc-xK-7pR"; */
+"Cmc-xK-7pR.text" = "Socket Mobile\nVersion 0.8";
```

Comment:
Is it a fix version here?

Reply:
This version text is updated on runtime by getting info from SDK and info.plist.